docs/mega-linter-runner.md

```diff
@@ -56,14 +56,10 @@ Sample `.pre-commit-config.yaml`:
 ```yaml
 repos:
   - repo: https://github.com/oxsecurity/megalinter
-    rev: v6.6.0 # Git tag specifying the hook, not mega-linter-runner, version
+    rev: v5.12.0 # Git tag specifying the hook, not mega-linter-runner, version
     hooks:
-      - id: megalinter-incremental # Faster, less thorough
-        stages:
-          - commit
-      - id: megalinter-full # Slower, more thorough
-        stages:
-          - push
+      - id: megalinter # Faster, less thorough, runs pre-commit by default
+      - id: megalinter-all # Slower, more thorough, runs pre-push by default
```

Comment:
@nvuillam, did the automation revert this portion of #1756 by accident?

Reply:
![image](https://user-images.githubusercontent.com/17500430/188266746-4bb68baa-25a2-4d2d-9b67-299079810266.png)

Meaning you need to update the README of mega-linter-runner so it is propagated to docs/mega-linter-runner.md :)

Reply:
You can do that in your current pre-commit PR ^^

Reply:
Wow, I should know that by now. Thank you for the reminder! I updated my PR accordingly.